templates/compositions/ec-component-navigation-links/navigation-links.html.twig

```diff
@@ -0,0 +1,69 @@
+{% spaceless %}
+
+{#
+  /**
+   * @file
+   * Theme implementation for ECL Navigation Links.
```

Comment:
Where is this naming come from? The ECL component is called "Navigation list" and in EWCMS we have also Navigation links that are doing something else. Or those will have similar purpose?  Also business calls this as Navigation blocks, and our epic is also called Navigation blocks. I am confused.

Reply:
Same for the path and file names across.